aad/spring-cloud-azure-starter-active-directory/aad-resource-server-obo/pom.xml

```diff
@@ -36,6 +35,11 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-web</artifactId>
     </dependency>
+    <!-- This is for using WebClient. -->
```

Comment:
Change this to <!-- This application is still a Web MVC application. Including the spring-boot-starter-webflux only to use the WebClient. -->